src/components/Board.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import Square from './Square'
+import lines from './winning-conditions'
+import Confetti from 'react-dom-confetti'
+import confettiConfig from './confetti-config'
+
+//Board props
+type BoardType = {
+  squares: string[][]
+  updateHistory: (squares: string[]) => void
+  currTurn: string
+}
+
+//component
+const Board: React.FC<BoardType> = (props: BoardType) => {
+  const [boardState, setBoardState] = useState<string[]>(props.squares[0])
+  const [isWinner, setIsWinner] = useState<boolean>(false)
+
+  /**
+   * Handles updating the sate of the board string array
+   * given a value from a square component
+   * @param id
+   * @param value
+   */
+  function handleClick(id: number, value: string) {
```

Comment:
I'd prepend on- for any event handlers like this

```suggestion
  const onSquareClick = (id: number, value: string) => {
```